aws/docker/worker/generate_profile_plot.py

```diff
@@ -63,22 +103,67 @@ def get_erddap_data(dataset_id):
     :return: pandas DataFrame with deployment variable values
     '''
     e = ERDDAP(
-        server='https://gliders.ioos.us/erddap',
+        server='http://gliders.ioos.us/erddap',
         protocol='tabledap',
-    )
+             )
+    info_url = e.get_info_url(dataset_id=dataset_id, response='csv')
+    info = pd.read_csv(info_url)
+
+    n_info = info.loc[info['Attribute Name'] == '_CoordinateAxisType']
+    coords_var = n_info['Variable Name'].values
+
+    v_info = info.loc[info['Attribute Name'] == 'standard_name']
+
+    # prepare list of science variables for plotting
+    sci_vars = []
+    # update PARAMETERS dictionary:
+    # (1) replace the parameter name by the one used in the data file.
+    old_par = []
+    new_par = []
+    # (2) delete parameters that are not found in the data files
+    del_par = []
+
+    for parameter in PARAMETERS:
+        sn_var = PARAMETERS[parameter]['standard_name']
+        vv_info = v_info.loc[v_info['Value'] == sn_var]
+        if len(vv_info) != 0:
+            try:
+                print('try', parameter)
```

Comment:
can you remove the prints? You can use the logging module if you want to log these messages